src/main/java/com/world/entertainment/worldentertainment/repository/CustomerRepository.java

```diff
@@ -0,0 +1,8 @@
+package com.world.entertainment.worldentertainment.repository;
+
+import com.world.entertainment.worldentertainment.entity.Customer;
+import org.springframework.data.repository.CrudRepository;
+
+public interface CustomerRepository extends CrudRepository<Customer, Integer> {
+
+}
```

Comment:
Add empty line in the end of this class. It's code style. We need add empty line after class body